L1Trigger/TrackFindingTracklet/python/Tracklet_cfi.py

```diff
@@ -13,6 +13,7 @@
                                                asciiFileName = cms.untracked.string(""),
                                                Extended = cms.bool(False),
                                                Reduced = cms.bool(False),
+                                               InventStubs = cms.bool(False),
```

Comment:
Is there a reason this parameter needs to be in Tracklet_cfi.py, and not just specified in Settings.h?